generators/app/templates/webapp/index.html

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta http-equiv="Cache-control" content="no-cache, no-store, must-revalidate">
-    <meta http-equiv="Pragma" content="no-cache">
-	<meta http-equiv="expires" content="0"> 
-	
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>UI5 Application <%= application %></title>
+	<head>
+		<meta http-equiv="Cache-control" content="no-cache, no-store, must-revalidate" />
+		<meta http-equiv="Pragma" content="no-cache" />
+		<meta http-equiv="expires" content="0" />
 
-	<script id="sap-ui-bootstrap"
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
erm.... what's that for? The App control will anyway set this!

Reply:
Ack